### ISSUE TITLE
Integration specs fail with ruby 2.3

### ISSUE THREAD

**Issue by esad** [realm/jazzy#516]
Getting the following diff when I run integration specs with ruby 2.3:

![screen shot 2016-04-06 at 19 07 03](https://cloud.githubusercontent.com/assets/11807/14327175/7ae4aea0-fc31-11e5-9e42-37f62e92da27.png)

It looks like behavior of some sorting method that is used has changed

**Comment by jpsim** [comment]
Thanks for reporting!

**Comment by segiddins** [comment]
@jpsim that looks like a swift version issue to me, since that's how swift's mangled type names have changed in between versions

**Comment by jpsim** [comment]
@segiddins I thought that too at first, but if you look closely, the declarations are identical, just sorted differently, thus a ruby issue as reported.

**Comment by segiddins** [comment]
ah. I didn't see that. Didn't we fix a related issue about stable sorting recently?

**Comment by jpsim** [comment]
Yes that was @esad in #474.

**Comment by istx25** [comment]
Any progress on this?

**Comment by jpsim** [comment]
> Any progress on this?

No, solving this amounts to finding a way to sort consistently across Ruby versions.
